src/sidebars/search/search.less

```diff
@@ -74,6 +78,17 @@
   .compact-search {
     display: block;
   }
+  .compact-search.expanded {
+    icon {
+      display: block;
+      #close-button {
+        position: relative;
+        float: right;
+        top: -16px;
+        right: 7px;
```

Comment:
True, those icons might be a special case, though, due to their window 'control-ey' nature. I'll merge it in anyway, and it can be something for later if UX picks it up.